cypress/e2e/8-motions.js

```diff
@@ -178,9 +181,70 @@ describe('User can create motions via UAC', () => {
     });
   });
 
+  it('Claiming Stakes', () => {
+    cy.login();
+    cy.visit(`/colony/${colonyName}`);
+
+    // Get amount of staked tokens
+    cy.getBySel('tokenActivationButton', { timeout: 12000 }).click();
+
+    // Get amount of staked tokens
+    cy.getBySel('stakedTokens', { timeout: 600000 })
+      .invoke('text')
+      .as('initialStakedTokens');
+
+    cy.getBySel('stakesTab').click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('claimableMotionsList', { timeout: 20000 })
+      .wait(2000) // Wait is required to ensure hash is included
+      .find(`[data-test="goToMotion"]`)
+      .first()
+      .click();
+
+    // Get the staked value being claimed
+    // We need a small wait to more reliably get the stakeBeingClaimed
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('stakedValue', { timeout: 60000 })
+      .wait(2000)
+      .invoke('text')
+      .as('stakeBeingClaimed');
+
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('claimStakeButton', { timeout: 20000 }).click().wait(15000);
+
+    // to close the gas station
+    cy.getBySel('actionHeading').click();
+
+    // Check that the active tokens are correct
+    cy.getBySel('tokenActivationButton', { timeout: 12000 }).click();
+
+    // function is required for `this` object to work
+    cy.getBySel('stakedTokens', { timeout: 6000 })
+      .invoke('text')
+      .as('newStakedTokens')
+      .then(function () {
```

Comment:
I thought about it and I don't like this unnamed function and `this` business. You don't need it. You cana just do `cy.get` with the alias and then use it in another `then` function. It will be more readable and it seems a preferable way of using aliases for cypress.
Please, let me knwo if it's not clear and I can explain on the call.

Reply:
I agree, I think it came out a lot better with a refactor, thank you for making this point.